plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -7529,40 +8097,55 @@ table.bar_chart {
 	}
 }
 
+@media only screen and (max-width: 1280px) {
+	#woocommerce-product-data,
+	#woocommerce-coupon-data {
+		.options_group,
+		// General coupos data does not have options group wrapper, make it look like it has.
+		&#general_coupon_data {
+			padding: $gap calc($gap * 4 / 3);
+		}
+	}
+}
+
 /**
   * Optimisation for screens 900px and smaller
   */
 @media only screen and (max-width: 900px) {
-	#woocommerce-coupon-data ul.coupon_data_tabs,
-	#woocommerce-product-data ul.product_data_tabs,
-	#woocommerce-product-data .wc-tabs-back {
-		width: 10%;
-	}
+	#woocommerce-product-data,
+	#woocommerce-coupon-data {
+		.panel-wrap {
+			grid-template-columns: [nav] 10% [content] 90%;
+		}
 
-	#woocommerce-coupon-data .wc-metaboxes-wrapper,
-	#woocommerce-coupon-data .woocommerce_options_panel,
-	#woocommerce-product-data .wc-metaboxes-wrapper,
-	#woocommerce-product-data .woocommerce_options_panel {
-		width: 90%;
+		.options_group,
+		// General coupos data does not have options group wrapper, make it look like it has.
+		&#general_coupon_data {
```

Comment:
The compiled selector `#woocommerce-coupon-data#general_coupon_data` seems to be incorrect.

![image](https://github.com/user-attachments/assets/cf3e3675-e371-46f0-8af0-e2013d65350f)

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/52632/commits/150f2be35a55eafcbe60721ba51315e72b5e90c4